ansible/playbooks/AdoptOpenJDK_Unix_Playbook/main.yml

```diff
@@ -29,6 +29,7 @@
     - NTP_TIME
     - gcc_48
     - gcc_7
+    - {role: nasm, when: ansible_architecture == 'x86_64'}
```

Comment:
Is this intended to take effect on OSX machines as well?

Reply:
I think the answer is "not yet". We haven't tried it on OSX. That being said, I know this package is related to building OpenJ9 on OSX.

Reply:
Afaik only required on x86.

Reply:
To be fair, we currently have plenty of other stuff in the playbooks which won't run on macos/x64 so I'm not too concerned about this one (Will need a major cleanup at some point though!)